.github/workflows/jacktrip.yml

```diff
@@ -710,3 +711,155 @@ jobs:
           draft: true
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  deploy_jtl:
+    if:  startsWith(github.ref, 'refs/tags/') && github.event_name == 'release' && needs.check-secrets.outputs.should_run == 'true'
+    needs: [build, sign]
+    runs-on: ${{ matrix.runs-on }}
+    name: ${{ matrix.name }}
+    strategy:
+      matrix:
+        include:
+          - name: Update Windows manifests
+            if: needs.check-secrets.outputs.should_run == 'true'
+            platform-name: win
+            release-name: Windows-x64
+            build-job-name: Windows-x64-qmake-gcc-static-bundled_rtaudio
+            runs-on: ubuntu-latest
+          - name: Update Mac manifests
+            if: needs.check-secrets.outputs.should_run == 'true'
+            platform-name: mac
+            release-name: macOS-x64
+            build-job-name: macOS-x64-qmake-clang-static-bundled_rtaudio
+            runs-on: ubuntu-latest
+          - name: Update Linux manifests
+            if: needs.check-secrets.outputs.should_run == 'true'
+            platform-name: linux
+            release-name: Linux-x64
+            build-job-name: Linux-x64-qmake-gcc-static
+            runs-on: ubuntu-latest
+    permissions:
+      contents: "read"
+      id-token: "write"
+    env:
+      DOWNLOAD_PATH: ${{ github.workspace }}/Install
+      GCP_IDP: ${{ secrets.GCP_IDP }}
+      GCS_SVC_ACCT: ${{ secrets.GCS_SVC_ACCT }}
+    steps:
+      - name: set version string for manifests
+        shell: bash
+        id: set-version
+        run: |
+          if [[ "$GITHUB_REF" == refs/tags/* ]]; then
+            VERSION=${GITHUB_REF#refs/tags/}
+          else
+            VERSION=${GITHUB_SHA::7}
+          fi
+          if [[ -n "${{ matrix.release-name }}" && "$GITHUB_REF" == refs/tags/* ]]; then
+            NAME="${{ matrix.release-name }}"
+          else
+            NAME="${{ matrix.build-job-name }}"
+          fi
+          if [[ "${{ matrix.platform-name }}" == "mac" ]]; then
+            EXTENSION=".pkg"
+            TYPE="signed-installer"
+          elif [[ "${{ matrix.platform-name }}" == "win" ]]; then
+            EXTENSION=".msi"
+            TYPE="signed-installer"
+          else
+            EXTENSION=".zip"
+            TYPE="binary"
+          fi
+          echo "version=$VERSION" >> $GITHUB_OUTPUT
+          echo "name=$NAME" >> $GITHUB_OUTPUT
+          echo "extension=$EXTENSION" >> $GITHUB_OUTPUT
+          echo "type=$TYPE" >> $GITHUB_OUTPUT
+      - name: download artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: ${{ env.DOWNLOAD_PATH }} # no "name" parameter - download all artifacts
```

Comment:
NIT but this "no name" comment doesn't apply to this section

Reply:
ah yeah